### PR TITLE
Ran clang-format version 17.0.6 for each style...

### PR DESCRIPTION
**Pull request by joelbenway**
...gnu, llvm, and microsoft styles have all evidently changed. Here are the differences.

### CODE REVIEW

File: gnu.cpp [hunk 1]
```diff
@@ -30,8 +30,7 @@ class Cowboy
   int Age;
 };
 
-Cowboy::Cowboy ()
-    : Age (45), GunCount (2)
+Cowboy::Cowboy () : Age (45), GunCount (2)
 {
```
Comment (motine) [inline gnu.cpp:34]: mmm, i tried to find the change in the coding standard, but could not find it. can you help me out an provide a reference?

File: llvm.cpp [hunk 2]
```diff
@@ -20,8 +20,7 @@ class Cowboy {
   int Age;
 };
 
-Cowboy::Cowboy()
-    : Age(45), GunCount(2) {
```
Comment (motine) [inline llvm.cpp:24]: i think i found the change [here](https://llvm.org/docs/CodingStandards.html#anonymous-namespaces)

File: microsoft.cpp [hunk 3]
```diff
@@ -29,8 +29,7 @@ class Cowboy
     int Age;
 };
 
-Cowboy::Cowboy()
-    : Age(45), GunCount(2)
```
Comment (motine) [inline microsoft.cpp:33]: mmm, can you provide a reference here too?

**Review by motine — COMMENTED** [review]
thanks @joelbenway for the proposed changes. please checkout my comments.

File: llvm.cpp (hunk 2 above)
Comment (joelbenway) [reply llvm.cpp:24]: That's the coding standard but I don't see the relevant instruction to support either style. But I didn't make these changes based on any published style guide.

The changes in this PR come from simply running `clang-format --style=llvm llvm.cpp` and seeing what changed. I did this for all the styles that clang-format provides as options. I'm assuming that the seven formats baked into the tool are authoritative for your purposes here. If this is the case I suggest verifying the command yourself with a recent version of clang-tools. If they are not, I apologize we should close this PR.

Comment (motine) [reply llvm.cpp:24]: ah no worries. i was just lacking the context where the changes came from. thanks for putting in the work!

File: microsoft.cpp (hunk 3 above)
Comment (motine) [reply microsoft.cpp:33]: addressed in comment above

**Pull request merged**